testData/deviceReporter/acceptanceTests/chrome.json

```diff
@@ -1,5 +1,9 @@
 [
     {
         "id": "com.ilunion.cloud4chrome"
```

Comment:
I think this would be a suitable pull request for removing all the old plugin stuff (it's been due for removing a loooong time)... Would you mind doing so; i.e. remove the entries from the solutions registry, device reporter file, tests, preferences sets. Anything you reuse - like this file, I think should be renamed to "browserExtension" or uioPlus instead of 'chrome', to be clear what it is related to